README.md

```diff
@@ -1,5 +1,27 @@
+# PMEM-CSI for Kubernetes
+
+<h2>*Note: This is Alpha code and not production ready.*</h2>
+
+Intel PMEM-CSI is a storage driver for container orchestrators like
+Kubernetes. It makes local persistent memory
+([PMEM](https://pmem.io/)) available as a filesystem volume to
+container applications. It can currently utilize non-volatile memory
+devices that can be controlled via the [libndctl utility
+library](https://github.com/pmem/ndctl). In this readme, we use
+*persistent memory* to refer to a non-volatile dual in-line memory
+module (NVDIMM).
+
+The [v0.6.0release](https://github.com/intel/pmem-csi/releases/tag/v0.6.0)
```

Comment:
A space between "v0.6.0" and "release" got lost here.

Reply:
Thx